worlds/Arena.wbt

```diff
@@ -476,6 +476,8 @@ DEF SHIP Transform {
         Shape {
           appearance PBRAppearance {
             roughness 1
+            metalness 0
+            baseColor 1 0 1
```

Comment:
Question: Are we concerned about contrast between tokens and the raised areas? Removing `metalness` does make them darker which helps with contrast, but still.